faq/pathogen-subs-guide.rst

```diff
@@ -263,58 +263,60 @@ for permitted values.
 Chromosome list file
 ''''''''''''''''''''
 
-The chromosome list file is an optional file for a complete assembly which describes the 'chromosomes' within
-the assembly. Chromosome in the context of ENA submissions means a range of complete replicons, as explained `here <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_
-and is used when describing a completed assembly.
-
-The chromosome list file is a tab separated file up to four columns. Each row describes each replicon unit within the assembly.
-Please refer to the `chromosome list file guide <https://ena-docs.readthedocs.io/en/latest/submit/fileprep/assembly.html#chromosome-list-file>`_
+The **chromosome list file** must be provided when the submission contains assembled chromosomes. This is a tab separated file up to four columns. Each row describes each replicon unit within the assembly. Please refer to the `chromosome list file guide <https://ena-docs.readthedocs.io/en/latest/submit/fileprep/assembly.html#chromosome-list-file>`_
 for permitted values.
 
+Note that ‘chromosome’ should here be understood as a general term for a range of complete replicons, including chromosomes of eukaryotes, prokaryotes, 
+and viruses, as well as organellar chromosomes and plasmids. All of these may be submitted within the same chromosome-level assembly.
+
+If your assembly is not complete, you can submit a **contig** or **scaffold** level assembly. The `Assembly levels <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_ tutorial page explains the assembly levels recognised by ENA.
+
 **chromosome_list.txt** examples
```

Comment:
Can this be more like a small subheading? Would apply to `manifest.txt example` above too. They get a bit lost on the page.

Reply:
The assembly submissions text has been reviewed and re-organised to make more sense